lab-ron/app/component/app-edit-gallery-item-form/app-edit-gallery-item-form-directive.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const angular = require('angular');
+angular.module('demoApp')
+.directive('appEditGalleryItemForm', function(){
+  return{
+    restrict: 'E',
+    replace: true,
+    scope: {
+      title: '=',
```

Comment:
Why are these ones done w/ '=', whereas the other ones are done w/ '@'?

Passing individual properties through is going to make things harder in the long run. It'll be easier on you if you pass the whole image object through.